zhaquirks/sonoff/s60zbtpf.py

```diff
@@ -0,0 +1,150 @@
+            id=0x7007,
+            type=t.uint8_t,
+        )
+        ac_current_max_overload_enable = ZCLAttributeDef(
+            name="ac_current_max_overload_enable",
+            id=0x700C,
+            type=t.uint8_t,
+        )
+        ac_voltage_max_overload_enable = ZCLAttributeDef(
+            name="ac_voltage_max_overload_enable",
+            id=0x700E,
+            type=t.uint8_t,
+        )
+        ac_power_max_overload_enable = ZCLAttributeDef(
+            name="ac_power_max_overload_enable",
+            id=0x7010,
+            type=t.uint8_t,
+        )
+        ac_current_max_overload = ZCLAttributeDef(
+            name="ac_current_max_overload",
+            id=0x700D,
+            type=t.uint32_t,
+        )
+        ac_voltage_max_overload = ZCLAttributeDef(
+            name="ac_voltage_max_overload",
+            id=0x700F,
+            type=t.uint32_t,
+        )
+        ac_power_max_overload = ZCLAttributeDef(
+            name="ac_power_max_overload",
+            id=0x7011,
+            type=t.uint32_t,
+        )
+
+    class ServerCommandDefs(zcl_f.BaseCommandDefs):
+        """Server command definitions."""
+
+        self_test: Final = zcl_f.ZCLCommandDef(
+            id=0x00, schema={"identify_time": t.uint8_t}, direction=False
+        )
+
+
+(
+    QuirkBuilder("SONOFF", "S60ZBTPF")
+    .replaces(SonoffCluster)
+    .switch(
+        SonoffCluster.AttributeDefs.outlet_control_protect_setting.name,
+        SonoffCluster.cluster_id,
+        off_value=0,
+        on_value=1,
+        translation_key="outlet_control_protect_setting",
+        fallback_name="Outlet control protect setting",
+    )
+    .switch(
+        SonoffCluster.AttributeDefs.ac_current_max_overload_enable.name,
+        SonoffCluster.cluster_id,
+        off_value=0,
+        on_value=1,
+        translation_key="ac_current_max_overload_enable",
+        fallback_name="AC current max overload enable",
+    )
+    .switch(
+        SonoffCluster.AttributeDefs.ac_voltage_max_overload_enable.name,
+        SonoffCluster.cluster_id,
+        off_value=0,
+        on_value=1,
+        translation_key="ac_voltage_max_overload_enable",
+        fallback_name="AC voltage max overload enable",
+    )
+    .switch(
+        SonoffCluster.AttributeDefs.ac_power_max_overload_enable.name,
+        SonoffCluster.cluster_id,
+        off_value=0,
+        on_value=1,
+        translation_key="ac_power_max_overload_enable",
+        fallback_name="AC power max overload enable",
+    )
+    .number(
+        SonoffCluster.AttributeDefs.ac_current_max_overload.name,
+        SonoffCluster.cluster_id,
+        ClusterType.Server,
+        1,
+        0.1,
+        17.0,
+        0.1,
+        unit=UnitOfElectricCurrent.AMPERE,
+        multiplier=0.001,
+        translation_key="ac_current_max_overload",
+        device_class=NumberDeviceClass.CURRENT,
+        fallback_name="AC current max overload",
+    )
```

Comment:
Let's explicitly specify what parameter we provide by using keyword arguments, so:
```suggestion
    .number(
        SonoffCluster.AttributeDefs.ac_current_max_overload.name,
        SonoffCluster.cluster_id,
        cluster_type=ClusterType.Server,
        endpoint_id=1,
        min_value=0.1,
        max_value=17.0,
        step=0.1,
        unit=UnitOfElectricCurrent.AMPERE,
        multiplier=0.001,
        translation_key="ac_current_max_overload",
        device_class=NumberDeviceClass.CURRENT,
        fallback_name="AC current max overload",
    )
```
Please also apply that to all other quirks v2 methods.

Omitting the parameter name for `attribute_name` and `cluster_id` is fine (first two parameters), as that's a common pattern across all our v2 quirks. We should explicitly provide all other names though.
In the future, we'll hopefully enforce this in the zigpy quirks v2 API.

Reply:
ok,I have added it